examples/Example/Program.fs

```diff
@@ -83,41 +84,40 @@ let app =
           | None ->
             store.set "counter" 1
             >>= OK "First time")
-    basic_auth // from here on it will require authentication
-    GET >>= url "/events" >>= request (fun r -> EventSource.hand_shake (CounterDemo.counter_demo r))
-    GET >>= browse' //serves file if exists
-    GET >>= dir' //show directory listing
+    basicAuth // from here on it will require authentication
+    GET >>= url "/events" >>= request (fun r -> EventSource.handShake (CounterDemo.counterDemo r))
+    GET >>= browseHomeDirectory //serves file if exists
```

Comment:
We'd have to change all of these:

    val browseHomeDirectory : WebPart
    val browseFileInHomeDirectory : fileName:string -> WebPart
    val dirHomeDirectory : WebPart

to

    val browseHome : WebPart
    val browseFileInHome : fileName:string -> WebPart
    val dirHome : WebPart

or

    val browseHome : WebPart
    val browseFileHome : fileName:string -> WebPart
    val dirHome : WebPart

or

    val browseHomeDir : WebPart
    val browseFileHomeDir : fileName:string -> WebPart
    val dirHomeDir : WebPart


I think the second option is best, I've applied it, let me know what you'd like.